packages/web/design-system/components/dropdown.tsx

```
@@ -119,6 +119,20 @@ export const Dropdown = React.forwardRef<HTMLButtonElement, DropdownProps>(
     const searchInputRef = React.useRef<HTMLInputElement>(null)
     const optionRefs = React.useRef<(HTMLDivElement | null)[]>([])
 
+    // Clean up option refs when options change
+    React.useEffect(() => {
+      if (optionRefs.current.length !== filteredOptions.length) {
+        optionRefs.current = new Array(filteredOptions.length).fill(null)
+      }
+    }, [filteredOptions.length])
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)